### PR TITLE
Driver invocation

### LINKED ISSUE #26: driver invocation interface
Implement interface for executing operation on driver(s)

- IN: list of drivers
- IN: function
- IN: params
- IN: is_parallel (flag to trigger parallel execution)
- OUT: list with results

### PR DESCRIPTION
**Pull request by anton-sidelnikov**
Multiple driver invocation support
Reduced API calls
Fix issues with unsupported parameters in elbv2

Parallel execution will add later
Part of #26

### CODE REVIEW

**Review by lego963 — DISMISSED** [review]

**Review by enrrou — DISMISSED** [review]

File: octavia_proxy/api/drivers/elbv2/driver.py [hunk 1]
```diff
@@ -36,20 +36,25 @@ def loadbalancers(self, session, project_id, query_filter=None):
         if not query_filter:
             query_filter = {}
 
-        results = []
-
+        result = []
+        # OSC tries to call firstly this function even if
+        # requested one resource by id, but filter by id doesn't
```
Comment (gtema) [inline octavia_proxy/api/drivers/elbv2/driver.py:41]: "is not supported" and not "doesn't supported"

File: octavia_proxy/api/drivers/elbv3/driver.py [hunk 2]
```diff
@@ -61,12 +61,17 @@ def loadbalancers(self, session, project_id, query_filter=None):
         query_filter.pop('project_id', None)
 
         result = []
-
+        # OSC tries to call firstly this function even if
+        # requested one resource by id, but filter by id doesn't
```
Comment (gtema) [inline octavia_proxy/api/drivers/elbv3/driver.py:65]: same as above

**Review by irina-pereiaslavskaia — DISMISSED** [review]

File: octavia_proxy/api/v2/controllers/l7rule.py [hunk 3]
```diff
@@ -39,8 +40,13 @@ def __init__(self, l7policy_id):
                          [wtypes.text], ignore_extra_args=True)
     def get_one(self, id, fields=None):
         """Gets a single l7rule's details."""
+        pcontext = pecan_request.context
         context = pecan_request.context.get('octavia_context')
-        l7rule = self.find_l7rule(context, self.l7policy_id, id)
+        query_params = pcontext.get(constants.PAGINATION_HELPER).params
+        is_parallel = query_params.pop('is_parallel', False)
```
Comment (gtema) [inline octavia_proxy/api/v2/controllers/l7rule.py:46]: is it really possible we ever get this QP (same question valid everywhere)?

**Review by gtema — CHANGES_REQUESTED** [review]
looks reasonable, just few small comment nits inline

File: octavia_proxy/api/v2/controllers/l7rule.py (hunk 3 above)
Comment (anton-sidelnikov) [reply octavia_proxy/api/v2/controllers/l7rule.py:46]: only during direct call to api, i suppose, but in next pr, i will enable parallel execution by default.

**Review by gtema — APPROVED** [review]

**Pull request merged**